scripts/install_prerequisites.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash 
+
+
+sudo apt-get update
+ 
+sudo apt-get install git 
+
+sudo apt-get install unzip
+ sudo apt-get install default-jre
+
+sudo apt-get install curl python-setuptools python-dev python-pip 
+sudo pip install --upgrade pip
```

Comment:
Ah, sorry, my bad -- I found it here: https://pip.pypa.io/en/stable/user_guide/#only-if-needed-recursive-upgrade

I was looking at another page, which I think short-cutted it to -U.